eventually/tests/unittests/authentication/testforgetpassword.py

```diff
@@ -0,0 +1,114 @@
+"""
+Forget Password test module
+===========================
+"""
+
+import json
+
+from django.test import TestCase, Client
+
+from authentication.views import CustomUser
+from utils.jwttoken import create_token
+
+VALID_EMAIL_TOKEN = create_token({"user_id":100, 'email': 'john@email.com'}, 60 * 60)
+INVALID_EMAIL_TOKEN = create_token({"user_id":999, 'email': 'john@email.com'}, 60 * 60)
+INVALID_TOKEN = "eyJ1c2VyX2lkIjoxMDAsImVtYWlsIjoiam9obkBlbWFpbC5"
```

Comment:
maybe it is better to mock functions that have to create and handle tokens.

Reply:
There are only 3 constants, guess there's no need to mock functions that create and handle tokens.